examples/syscall.c

```diff
@@ -14,15 +14,25 @@
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/moduleparam.h> /* which will have params */
-#include <linux/syscalls.h>
-#include <linux/unistd.h> /* The list of system calls */
+#include <linux/unistd.h>      /* The list of system calls */
+#include <linux/version.h>
 
 /* For the current (process) structure, we need this to know who the
  * current user is.
  */
 #include <linux/sched.h>
 #include <linux/uaccess.h>
 
+/* The ksys_close in function aquire_sys_call_table is no longer supported
+ * after kernel 5.11, we need to replace it with close_fd instead.
+ */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0))
+#include <linux/syscalls.h> /* For ksys_close for backward compatibility */
```

Comment:
It was not intended to be "For ksys_close for backward compatibility." Instead, it should be
> ksys_close() wrapper for backward compatibility